lib/billing/request.ts

```diff
@@ -163,6 +163,8 @@ async function requestUsagePayment({
         name: plan,
         returnUrl,
         test: isTest,
+        trialDays: billingConfig.trialDays,
+        replacementBehavior: billingConfig.replacementBehavior,
```

Comment:
Just checking - is the actual API OK with these values being present but `null`? I would imagine so, but probably worth testing out.